v2/yarpcgrpc/outbound.go

```diff
@@ -95,10 +94,12 @@ func (o *Outbound) Call(ctx context.Context, req *yarpc.Request, reqBuf *yarpc.B
 		return nil, nil, err
 	}
 
+	appErr := metadataToApplicationError(invokeErr, responseMD)
+	errorInfo := yarpcerror.ExtractInfo(appErr)
 	return &yarpc.Response{
-			Peer:             metadataToPeer(responseMD),
-			Headers:          responseHeaders,
-			ApplicationError: metadataToApplicationError(invokeErr, responseMD),
+			Peer:                 metadataToPeer(responseMD),
+			Headers:              responseHeaders,
+			ApplicationErrorInfo: &errorInfo,
 		},
 		yarpc.NewBufferBytes(responseBody),
 		invokeErr
```

Comment:
It looks like we are losing information between the creation of the `appErr` and extracting the info adding it to the `Response`. Do we care about the information lost in `appErr`?